packages/ui-extensions/src/surfaces/checkout/api/standard/standard.ts

```diff
@@ -1756,7 +1756,29 @@ export interface VisitorConsent {
   saleOfData?: boolean;
 }
 
+export interface TrackingConsentMetafield {
+  /**
+   * The name of the metafield. It must be between 3 and 30 characters in
+   * length (inclusive).
+   */
+  key: string;
+  /**
+   * The information to be stored as metadata. If the value is `null`, the metafield will be deleted.
```

Comment:
We are using the term "metadata" and "metafield" interchangeably here, but that may cause some confusion.

Reply:
Ok, i'm fine keeping this as is. We can always revisit it.